plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/actionworkers/KubernetesClusterResourceModifierActionWorker.java

```diff
@@ -511,6 +493,7 @@ protected FirewallRule removeApiFirewallRule(final IpAddress publicIp) {
                     firewallRule.getSourcePortEnd() == CLUSTER_API_PORT) {
                 rule = firewallRule;
                 firewallService.revokeIngressFwRule(firewallRule.getId(), true);
+                logger.debug(String.format("The API firewall rule [%s] with the id [%s] was revoked",firewallRule.getName(),firewallRule.getId()));
```

Comment:
No need to use `String.format()` here, with the updated log4j.

Reply:
see https://github.com/apache/cloudstack/pull/8801#discussion_r1639482249